client/pk-monitor.c

```diff
@@ -128,6 +128,42 @@ pk_monitor_adopt_cb (PkClient *_client, GAsyncResult *res, gpointer user_data)
 	}
 }
 
+static gchar*
+pk_monitor_get_caller_info (GDBusProxy *bus_proxy, const gchar *name)
+{
+	gboolean ret;
+	gchar *cmdline = NULL;
+	g_autofree gchar *filename = NULL;
+	g_autoptr(GError) error = NULL;
+	guint pid = G_MAXUINT;
+	g_autoptr(GVariant) value = NULL;
+
+	/* get pid from DBus */
+	value = g_dbus_proxy_call_sync (bus_proxy,
+					"GetConnectionUnixProcessID",
+					g_variant_new ("(s)",
+						       name),
+					G_DBUS_CALL_FLAGS_NONE,
+					2000,
+					NULL,
+					&error);
+	if (value == NULL) {
+		g_warning ("Failed to get pid for %s: %s",
+			   name, error->message);
+		return g_strdup_printf ("bus:%s", name);;
```

Comment:
```suggestion
		return g_strdup_printf ("bus:%s", name);
```

(accidental double semicolon)